pkgs/development/libraries/gtk/3.x.nix

```diff
@@ -70,13 +71,6 @@ stdenv.mkDerivation rec {
       url = "https://bug757142.bugzilla-attachments.gnome.org/attachment.cgi?id=344123";
       sha256 = "0g6fhqcv8spfy3mfmxpyji93k8d4p4q4fz1v9a1c1cgcwkz41d7p";
     })
-
-    # Fix path handling in pkg-config
```

Comment:
this commit never made it into this tag https://gitlab.gnome.org/GNOME/gtk/-/commit/6d9db8610eff8c12d594d53b7813d9eea1247801. We still need it and should PR upstream to backport it

Reply:
:+1:.

Reply:
this fixes https://github.com/NixOS/nixpkgs/pull/98316#issuecomment-695811478

Reply:
It still applies.

Reply:
Yep, that make backporting very easy. Opened https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/2605